changelog.yaml

```diff
@@ -56,6 +56,19 @@
   date: 2023-10-06
   changes:
 
+    - type: enhancement
+      impact: minor
```

Comment:
The versions in the [Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) file needs to be adapted accordingly.

Reply:
Done in https://github.com/SAP/stewardci-core/pull/410/commits/ea3403ac1a8e2696997cbc5838232529e5b11e5c